packages/intellij-idea-community-bin/intellij-idea-community-bin.pacscript

```diff
@@ -0,0 +1,39 @@
+name="intellij-idea-community-bin"
+pkgver="2023.3.6"
+pkgname="intellij-idea-community"
+gives="intellij-idea-community"
+repology=("project: intellij-idea-community")
+maintainer="Gianluca Mastrolonardo <gianlucamastrolonardo10@gmail.com>"
+pkgdesc="IntelliJ IDEA is an IDE for developing computer software written in Java, Kotlin, Groovy, and other JVM-based languages"
+arch=('amd64' 'arm64')
+if [[ ${CARCH} == amd64 ]]; then
+  url="https://download-cdn.jetbrains.com/idea/ideaIC-${pkgver}.tar.gz"
+  hash="5e376677d4b9810f4171700ce9d9d3ece58f245ce8d9f2011b05966df5dd7ddd"
+else
+  url="https://download-cdn.jetbrains.com/idea/ideaIC-${pkgver}-aarch64.tar.gz"
+  hash="cfc98c4a09f6740ef47a3dbc8f2e71a66a8aab2eae3fb1b901c2db5dcce5f060"
+fi
+repology=("project: intellij-idea-community")
+desktop_file_sum="20c489d224f83cf4b29174b8c398e256b1df0e9cb0b920890f39c5796404b293"
+
+prepare() {
+  wget -qO "intellij-idea-community.desktop" "https://raw.githubusercontent.com/Rockstar9604/pacstall-programs/master/packages/${name}/intellij-idea-community.desktop"
```

Comment:
change this to our repo (even though the path doesn't exist yet) and I can merge it then

Reply:
You're right, sorry, I forgot to reciprocate the repo. I've fixed it.